conda/cli/activate.py

```diff
@@ -32,11 +35,27 @@ def prefix_from_arg(arg):
 
 
 def binpath_from_arg(arg):
-    path = join(prefix_from_arg(arg), 'bin')
-    if not isdir(path):
-        sys.exit("Error: no such directory: %s" % path)
+    if sys.platform == "win32":
+        path = [prefix_from_arg(arg),
+                join(prefix_from_arg(arg), 'Scripts'),
+                join(prefix_from_arg(arg), 'Library/bin'),
+               ]
+    else:
+        path = [join(prefix_from_arg(arg), 'bin')]
+    for p in path:
+        if not isdir(p):
+            sys.exit("Error: no such directory: %s" % p)
```

Comment:
This test is redundant after my fix. It's also not clear to me if exiting is the desired behavior here.